api/src/main/java/com/thesurvey/api/controller/AuthenticationController.java

```diff
@@ -19,6 +18,7 @@
 import io.swagger.annotations.ApiResponses;
 
 @Api(tags = "인증")
+@Validated
```

Comment:
What is difference using @Validated and @Valid annotation for each controllers?

Reply:
@Validated annotation can specify groups for validation of methods or classes. and This performs validation prior to method execution. If validation fails, a MethodArgumentNotValidException exception occurs.

@Valid annotation, on the other hand, performs validation of the fields in the object. It performs validation at the time when you create the parameter object. If validation fails, a ConstructViolationException exception is generated.

but since we don't use validation groups, I will revise whether these two annotations need to be used together.

Reply:
Thank you for this explanation!